src/bin/vip-logs.js

```diff
@@ -43,14 +42,89 @@ export async function getLogs( arg: string[], opt ): Promise<void> {
 
 	await trackEvent( 'logs_command_success', {
 		...trackingParams,
-		logs_output: logs.length,
+		total: logs.nodes.length,
 	} );
 
-	if ( ! logs.length ) {
+	if ( ! logs.nodes.length ) {
 		console.error( 'No logs found' );
 		return;
 	}
 
+	printLogs( logs.nodes, opt.format );
+}
+
+export async function followLogs( opt ): Promise<void> {
+	let after = null;
+	let isFirstRequest = true;
+	// How many times have we polled?
+	let requestNumber = 0;
+
+	const trackingParams = getBaseTrackingParams( opt );
+
+	// Set an initial default delay
+	let delay = DEFAULT_POLLING_DELAY_IN_SECONDS;
+
+	while ( true ) {
+		const limit = isFirstRequest ? opt.limit : LIMIT_MAX;
+
+		requestNumber++;
+		trackingParams.request_number = requestNumber;
```

Comment:
[MINOR] I'm wondering if it's worth adding a unique identifier (uuid maybe) in case we want to easily filter a sequence of calls.

Reply:
Sounds interesting... Not sure if this kind of filtering can be easily done in the tracking functionality, maybe @mjangda  knows better.

In any case, I think I'm merging this now so I can release the testing version, we may include this change in another PR